test/integration/config.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/sethvargo/go-envconfig"
+)
+
+type config struct {
+	ProjectID                 string        `env:"INTEG_TEST_PROJECT_ID,required"`
+	IAMExpirationStartTime    string        `env:"INTEG_TEST_IAM_EXPIRATION_START_TIME,required"`
+	IAMExpirationDurationHour string        `env:"INTEG_TEST_IAM_EXPIRATION_DURATION_HOUR,required"`
+	IAMExpiration             string        `env:"INTEG_TEST_IAM_EXPIRATION,required"`
```

Comment:
Do these really need to be configurable?

Reply:
Yeah, maybe it is a bit earlier now, we would want the expiration for cleanup afterwards in the workflow, the start time and duration hour are used to calculate the expiration.

Reply:
To remove an IAM binding it needs the entire condition including expiration expression and title.